samples/sqltransport-nhpersistence-outbox-ef/Version_2.1/Receiver/Program.cs

```diff
@@ -7,6 +7,8 @@ class Program
 {
     static void Main()
     {
+        Console.Title = "Receiver";
```

Comment:
While i dont object to this change it now makes it inconsistent with all the other samples. If u want to do this then do do a new pr that just does this and then document it as the recommended  approach for writing samples